pages/admin/programs/index.tsx

```diff
@@ -0,0 +1,183 @@
+import { AdminEmptyState } from "@components/admin/AdminEmptyState";
+import { useState } from "react";
+
+type BrowseProgramsProps = {
+    session: Record<string, unknown>;
+};
+
+const Links = [
+    { name: "Add Program", url: "/admin/program/create" },
+    { name: "Add Class", url: "/admin/class/create" },
+];
+
+const NavLink = ({ href, children }: { href: string; children: ReactNode }) => (
+    <Link href={href}>
+        <HStack>
+            <Button
+                fontSize="16px"
+                backgroundColor="white"
+                fontWeight="400"
+                px={"16px"}
+                py={"8px"}
+                border-radius="6px"
+                border="1px solid #0C53A0"
+            >
+                <SmallAddIcon mr="11px" />
+                {children}
+            </Button>
+        </HStack>
+    </Link>
+);
+
+export const BrowsePrograms: React.FC<BrowseProgramsProps> = (props) => {
+    const router = useRouter();
+    const [searchTerm, setSearchTerm] = useState("");
+
+    const {
+        programs: programCardInfos,
+        isLoading,
+        error,
+    } = usePrograms(router.locale as locale);
+
+    if (isLoading) {
+        return <Loading />;
+    } else if (error) {
+        return <Box>An Error has occurred</Box>;
+    }
+
+    const filteredCards = programCardInfos.filter((val) => {
+        if (searchTerm == "") {
+            return val;
+        } else if (
+            val.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+            val.description.toLowerCase().includes(searchTerm.toLowerCase())
+        ) {
+            return val;
+        }
+    });
+    return (
+        <Wrapper session={props.session}>
+            <Box>
+                <Box bg={"transparent"} px={"50px"} pt={"20px"} mx={"auto"}>
+                    <Flex h={"94px"}>
+                        <HStack spacing={8}>
+                            <Text
+                                fontSize="22px"
+                                fontWeight="bold"
+                                color={colourTheme.colors.Blue}
+                            >
+                                Programs
+                            </Text>
+                        </HStack>
+                        <Spacer />
+                        <HStack spacing={4}>
+                            {Links.map((linkInfo) => (
+                                <NavLink
+                                    key={linkInfo.name}
+                                    href={linkInfo.url}
+                                >
+                                    {linkInfo.name}
+                                </NavLink>
+                            ))}
+                        </HStack>
+                    </Flex>
+                </Box>
+                <Divider
+                    orientation="horizontal"
+                    marginTop="25px"
+                    marginBottom="30px"
+                    border="2px"
+                />
```

Comment:
we can reduce the margin bit to 24px
```suggestion
                <Divider orientation="horizontal" my={6} border="2px" />
```